### PR TITLE
[7.x] Add an exit code 1 when the command was not confirmed

### PR DESCRIPTION
**Pull request by williamdes**
Hi !
I recent change did make our pipeline pass but in fact nothing was executed.
**This is because the exit code is 0 instead of 1 or another > 0 code.**
I hope you will accept my contribution.
Also I can rebase on the right branch for this PR, I used 6.0 because I did not know if you would accept the change for previous versions.

image
image

### CODE REVIEW

File: src/Illuminate/Database/Console/Migrations/ResetCommand.php [hunk 1]
```diff
@@ -47,12 +47,12 @@ public function __construct(Migrator $migrator)
     /**
      * Execute the console command.
      *
-     * @return void
+     * @return int|void
```
Comment (GrahamCampbell) [inline src/Illuminate/Database/Console/Migrations/ResetCommand.php]: `int|void` is not a valid return type.

**Review by GrahamCampbell — CHANGES_REQUESTED** [review]

File: src/Illuminate/Database/Console/Migrations/ResetCommand.php (hunk 1 above)
Comment (williamdes) [reply src/Illuminate/Database/Console/Migrations/ResetCommand.php]: From what point of view ?

I would agree to add a `return 0;` and then use `@return int`
Is that what you would want ?

Comment (GrahamCampbell) [reply src/Illuminate/Database/Console/Migrations/ResetCommand.php]: > From what point of view ?

`void` cannot appear in a union type in PHP. https://wiki.php.net/rfc/union_types_v2

> I would agree to add a return 0; and then use @return int

Yes, that's the only thing we can do here. Please do that.

Comment (williamdes) [reply src/Illuminate/Database/Console/Migrations/ResetCommand.php]: Done in f59885b544c7b4516d68ee38a68a507816a0febf

Thank you for the documentation link :)

**Comment by williamdes** [comment]
@GrahamCampbell are the base and target branch the right ones ?

**Comment by GrahamCampbell** [comment]
I've updated the target to 7.x, since those commands have changed a bit in 7.x, and that this is really a breaking change. People may be relying on the fact that in 6.x, these commands silently do nothing.

Please could you rebase against `upstream/master`.

**Comment by williamdes** [comment]
Done as 6939128779e9a9d45405beddaa94ff430cf1a6ba

**Pull request merged**